docs/development/docrules.rst

```diff
@@ -61,7 +61,8 @@ function, class, or method definition. The docstring is a special attribute
 of the object (``object.__doc__``) and, for consistency, is surrounded by
 triple double quotes, i.e.::
 
-   """This is the form of a docstring.
+   """
+   This is the form of a docstring.
```

Comment:
Yes, I did not mean to imply that I did this *because* of PEP 8. Actually I think the only PEP 8 change in this is the spaces after the `#` comments.